tests/unit/create/test_creator.py

```diff
@@ -621,3 +622,16 @@ def test_pth_in_site_vs_PYTHONPATH(tmp_path):
         env=env,
     )
     assert out == "ok\n"
+
+
+def test_getsitepackages(tmp_path):
```

Comment:
`test_get_site_packages`

Reply:
I called it `test_getsitepackages` because the name of the python function is `getsitepackages`. Should I change it anyways?

Reply:
yes